frontend/apps/reader/modules/readerback.lua

```diff
@@ -1,6 +1,9 @@
+local _ = require("gettext")
```

Comment:
The symbol would go all the way at the bottom in this particular case, see https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires

Reply:
yes sorry, I forgot